spec/spec_helper.rb

```diff
@@ -46,24 +46,24 @@
     add_filter 'version.rb'
 
     # Modules
-    add_group 'Policies', 'app/policies'
-    add_group 'Serializers', 'app/serializers'
-    add_group 'Services', 'app/services'
-    add_group 'Swagger', 'app/swagger'
-    add_group 'Uploaders', 'app/uploaders'
     add_group 'AppealsApi', 'modules/appeals_api/'
     add_group 'AppsApi', 'modules/apps_api'
     add_group 'ClaimsApi', 'modules/claims_api/'
     add_group 'CovidVaccine', 'modules/covid_vaccine/'
+    add_group 'HealthQuest', 'modules/health_quest/'
+    add_group 'Mobile', 'modules/mobile/'
     add_group 'OpenidAuth', 'modules/openid_auth/'
+    add_group 'Policies', 'app/policies'
+    add_group 'Serializers', 'app/serializers'
+    add_group 'Services', 'app/services'
+    add_group 'Swagger', 'app/swagger'
+    add_group 'Uploaders', 'app/uploaders'
```

Comment:
Do you want Uploaders to come after TestUserDashboard?

Reply:
Wow, I clearly don't know the English alphabet lol. I had a merge conflict and must have missed the typos